src/domain/Phone/WebRTCPhone.js

```diff
@@ -22,19 +22,27 @@ export default class WebRTCPhone extends Emitter implements Phone {
 
   audioOutputDeviceId: string | typeof undefined;
 
+  audioRingDeviceId: string | typeof undefined;
+
   ringingEnabled: boolean;
 
   acceptedSessions: Object;
 
   rejectedSessions: Object;
 
-  constructor(client: WazoWebRTCClient, audioOutputDeviceId?: string, allowVideo: boolean = false) {
+  constructor(
+    client: WazoWebRTCClient,
+    audioOutputDeviceId?: string,
+    audioRingDeviceId?: string,
+    allowVideo: boolean = false,
```

Comment:
If you change the order or the parameter this will break the compatibility of previous version. Can you put `audioRingDeviceId` at the end ?

Reply:
if you ask nicely, sure (consider it done)